repairnator/scripts/launch_scanner.sh

```diff
@@ -16,16 +16,18 @@ function ca {
   fi
 }
 
-SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+if [ "$#" -eq 0 ]; then
```

Comment:
I don't understand this if...

Reply:
`launch_scanner.sh` doesn't require any arg. But if this is running throught `launch_repairnator.sh`, it can't execute those lines because the things were already initialized.

Reply:
See my answer above. Current design is really unclear.

Reply:
I agree.